CDTDatastoreTests/Encryption/TD_DatabaseEncryptionTests.m

```diff
@@ -17,12 +17,13 @@
 
 #import "CDTEncryptionKeyNilProvider.h"
 #import "CDTHelperFixedKeyProvider.h"
+#import "CloudantSyncTests.h"
 #import "CloudantTests+EncryptionTests.h"
 #import "FMDatabase+SQLCipher.h"
 
 #import "TD_Database.h"
 
-@interface TD_DatabaseEncryptionTests : CloudantTests
+@interface TD_DatabaseEncryptionTests : CloudantSyncTests
```

Comment:
It's a shame this needs to inherit from `CloudantSyncTests` just for `createTemporaryDirectoryAndReturnPath`.

It's slightly out of scope but I wonder whether we should consider moving this method into `CloudantTests`.